tests/integration/test_definitions.py

```diff
@@ -362,7 +382,7 @@ def define_tests():
         # provenence_equality_checks, # omitting provenance equality check because it is broken
         # style_checks, # omitting style checks due to different results on different machines
         dependency_checks,
-        # distributed_tests, # omitting distributed tests as they are not yet ready
+        distributed_tests
```

Comment:
does the ci run this or just the --local tests? b/c we for distributed we don't have a ci version working (until we get a rabbit and/or redis docker pop-up server in the pipeline)

Reply:
It now runs the distributed tests and also then fails out (because this doesn't setup the docker containers). I'm looking into how to set that up with Actions.

Reply:
The CI now stands up a docker image with a rabbitmq server, but it is not linked for the workflow to access, I'm still figuring out how to do that, but haven't spent any time on this since the last commit as other work's had priority. 

As a thought, this could be merged, with the CI only run against the local tests, and we'd just require any developer to run the full, distributed test suite against whatever cluster environment they developed on to verify nothing's been broken. That is not a long term solution, but 
A. it would only be a stopgap until I have time to come back and figure out the linking 
B. we've had a similar manual process as the working solution for black and isort until now (that said, I put up [this PR](https://github.com/LLNL/merlin/pull/336) to end that) 
C. most importantly, as-is this runs on LC machines, merging would enable testing against a distributed configuration, which is a critical and significant improvement, even if it isn't (yet) automated for the CI in our Actions environment.